databases/week1/homework.sql

```diff
@@ -0,0 +1,103 @@
+Select
+*
+FROM
+task;
+
+-- 1.Find out how many tasks are in the task table
+SELECT COUNT(id)
+    id
+FROM
+    task;
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id)
+    id
+FROM
+    task
+WHERE
+due_date IS NULL;
+
+-- 3.Find all the tasks that are marked as done
+SELECT
+    *
+FROM
+    task
+JOIN
+    status
+ON
+    task.status_id = status.id
+WHERE 
+    name LIKE "Done";
```

Comment:
WHERE
    status.name = "Done";
_Pointer_
Use Like operator for only pattern matches, use of Equals operator is more appropriate